Nickvision.Aura.Tests/Aura.Test.cs

```diff
@@ -0,0 +1,15 @@
+using Xunit.Abstractions;
+
+namespace Nickvision.Aura.Tests;
+
+public class AuraTest
```

Comment:
Since the class name is `AuraTest` could name the file `AuraTest.cs` instead of `Aura.Test.cs`